### PR TITLE
ADEN-8295 AE3 on search pages.

### PR DESCRIPTION
**Pull request by jacekbj**
Turn AE3 on on search pages.

Hide TOP_BOXAD slot on screen narrower than 1024.

### CODE REVIEW

File: includes/wikia/DefaultSettings.php [hunk 1]
```diff
@@ -973,6 +973,12 @@
  */
 $wgAdDriverAdEngine3EnabledOnFeaturedVideoPages = true;
 
+/**
+ * @name $wgAdDriverAdEngine3EnabledOnOasisSearchPages
+ * Enables AdEngine3 extension on search pages
+ */
+$wgAdDriverAdEngine3EnabledOnOasisSearchPages = true;
```
Comment (fraszczakszymon) [inline includes/wikia/DefaultSettings.php]: `false`? ;)

File: extensions/wikia/AdEngine3/src/slots.js [hunk 2]
```diff
@@ -27,6 +19,15 @@ function isIncontentBoxadApplicable() {
 		!context.get('wiki.targeting.wikiIsCorporate');
 }
 
+/**
+ * Enables top_boxad on screen with width >= 1024px.
+ *
+ * @returns {boolean}
+ */
+function isTopBoxadEnabled() {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/src/slots.js]: Can we keep the same convention as above? i.e. `isTopBoxadApplicable`
Or change above method to `*Enabled`

**Review by fraszczakszymon — APPROVED** [review]

File: includes/wikia/DefaultSettings.php (hunk 1 above)
Comment (jacekbj) [reply includes/wikia/DefaultSettings.php]: Fixed

File: extensions/wikia/AdEngine3/src/slots.js (hunk 2 above)
Comment (jacekbj) [reply extensions/wikia/AdEngine3/src/slots.js]: Fixed

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**